{"uuid": "9c21535e-47ba-409b-bfa0-8a9a37ed55cc", "counter": 5, "stdout": "\u001b[0;31mfatal: [10.104.10.20]: FAILED! => {\"changed\": false, \"module_stderr\": \"'PowerShell' is not recognized as an internal or external command,\\r\\noperable program or batch file.\\r\\n\", \"module_stdout\": \"\", \"msg\": \"MODULE FAILURE: No start of json char found\\nSee stdout/stderr for the exact error\", \"rc\": 1}\u001b[0m", "start_line": 4, "end_line": 5, "runner_ident": "e0befab0-7f88-4f2f-ae84-097232e84bb2", "event": "runner_on_failed", "pid": 229535, "created": "2025-05-17T23:20:54.370047+00:00", "parent_uuid": "005056b7-f67d-2a16-5630-000000000004", "event_data": {"playbook": "/tmp/tmpezo3r6kt.yml", "playbook_uuid": "e27907b7-219a-4ced-9e40-afe52100cbac", "play": "Pruebas b\u00e1sicas de conectividad y sistema en Windows", "play_uuid": "005056b7-f67d-2a16-5630-000000000002", "play_pattern": "target_host", "task": "Prueba de ping (win_ping)", "task_uuid": "005056b7-f67d-2a16-5630-000000000004", "task_action": "ansible.windows.win_ping", "resolved_action": "ansible.windows.win_ping", "task_args": "", "task_path": "/tmp/tmpezo3r6kt.yml:9", "host": "10.104.10.20", "remote_addr": "10.104.10.20", "res": {"module_stdout": "", "module_stderr": "'PowerShell' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n", "msg": "MODULE FAILURE: No start of json char found\nSee stdout/stderr for the exact error", "rc": 1, "_ansible_no_log": false, "changed": false}, "start": "2025-05-17T23:20:53.794604+00:00", "end": "2025-05-17T23:20:54.369629+00:00", "duration": 0.575025, "ignore_errors": null, "event_loop": null, "uuid": "9c21535e-47ba-409b-bfa0-8a9a37ed55cc"}}